.github/workflows/pull-request.yaml

```diff
@@ -55,9 +55,9 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           if [ -z "$GRADLE_MODULE" ]; then
-            ./gradlew --no-daemon test
+            ./gradlew --no-daemon build
```

Comment:
suggestion(non-blocking): This will cover more ground, i.e. we can check more stuff than linting, but it will likely be a bit slower.
An alternative could be to have a different `job`/step that runs the linter ([wallet PR setup](https://github.com/monta-app/service-wallet/blob/staging/.github/workflows/pull_request_checks.yml#L11)).